src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -97,7 +95,11 @@ def get_disk_device(self) -> str:
         raise NotImplementedError
 
     @abstractmethod
-    def _add_disk_to_board(self, disk_image: DiskImageResource) -> None:
+    def _add_disk_to_board(
+        self,
+        disk_image: DiskImageResource,
+        bench_image: Optional[DiskImageResource],
```

Comment:
The typing here is a little wrong, I believe it should be a list of `DiskImageResources`. You can see my other comment on this but I think the name should be `additional_disk_images`, and the docstring in this function should be updated to explain things should also be updated.

Reply:
Done!